src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
I definitely would prefer a secondary API. I think it was @tsullivan that was the holdout there. :)